app/models/search.rb

```diff
@@ -24,6 +24,7 @@ def results
     res = res.bot_author unless bot_author.nil?
     res = res.unlabelled unless unlabelled.nil?
     res = res.is_private(is_private) unless is_private.nil?
+    res = res.muted unless is_muted.nil? || !is_muted
```

Comment:
Does this work for `muted:false`, i.e `unmuted`?

Reply:
yes. for both `true` and `false` case

Reply:
`muted: false` case is the same as having null value for `muted_at` in DB. So I am only applying scope for `muted:true` case

Reply:
Added test case for `muted:false`